.env.example

```
@@ -1,9 +1,9 @@
-APP_NAME=interns2025c
-VITE_APP_NAME=interns2025c
+APP_NAME=LapGo
+VITE_APP_NAME=LapGo
 APP_ENV=local
 APP_KEY=base64:sCsJw8z+d/4ymp0OvzSip2h4Vp2hZZhpV2uOxgTqP94=
 APP_DEBUG=true
-APP_URL=http://interns2025c.blumilk.local.env
+APP_URL=http://LapGo.blumilk.local.env
```

```suggestion
APP_URL=http://lapgo.blumilk.local.env
```

Domain is not case sensitive.